### DISCUSSION TITLE
Let's talk about @okikio/animate

### DISCUSSION THREAD

**Discussion by okikio** [okikio/native#5] [Ideas]
How can [@okikio/animate](https://github.com/okikio/native/tree/master/packages/animate#readme) be improved?

**Comment by loops** [comment]
With all due respect, since you made some really cool stuff here, it would be helpful to put a prominent disclaimer that the project was mostly a learning exercise and is no longer being maintained.   It is quite unfortunate to spend time reading and learning about it, only to find out that it should not be used in any new projects.

**Reply by okikio** [reply]
Sorry about that, the project itself really never took off and I had less and less time to work on it over the past couple years, so I couldn't dedicate as much time to it, but I do have something in the works that will replace this project and potentially be more useful long term. I do appreciate you taking the time to explore the project.

> **Note**: Though I couldn't finish it in time, a number of complex features were in development in the `dev` branch that you may find interesting https://github.com/okikio/native/tree/dev/packages/animate

> **Note**: The complex easing portions of the project were later spun off into a separate package that seems to be more popular https://spring-easing.okikio.dev (https://github.com/okikio/spring-easing#readme)
